### PR TITLE
Ad block onboarding flow

### PR DESCRIPTION
**Pull request by chung-lai**
Day 0 ad block flow

* Enabled ad blocker by default
* Before user's first navigation, zero query shows ad blocker enabled by default
* On first ad blocked/cookie popup blocked, educational screen pop up once on screen to educate user
* Rename "Cookie Cutter" -> "Neeva Shield"

What's **NOT** included:

* Known issue on pop up resizing when toggle the switch on educational screen (non-blocker)
* Not behind experiment flag

https://user-images.githubusercontent.com/87332578/190699057-d55bda44-4fa3-4b83-a990-7ed0087ab8b5.mov

| Zero Query | ad blocked | deactivate | cookie popup blocked |
| --- | --- | --- | --- |
| Screen Shot 2022-09-16 at 10 20 30 AM | Screen Shot 2022-09-16 at 10 21 24 AM | Screen Shot 2022-09-16 at 10 20 50 AM | Screen Shot 2022-09-16 at 10 20 43 AM |

**Settings**

| Settings | Shield menu |
| --- | --- |
| Screen Shot 2022-09-16 at 10 23 02 AM | Screen Shot 2022-09-16 at 10 23 07 AM |

**iPad**
Screen Shot 2022-09-16 at 10 19 19 AM

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [x] Existing automated tests are enough to cover my changes

### CODE REVIEW

**Comment by hwiatneeva** [comment]
Thanks @chung-lai !

Seems like the strings and the icons refer back to a few days older version. Here's the latest from Sep 13, i.e. latest & final: 

* strings:  
** zero query subtext:  "Browse faster without spammy, screen-filling ads. Experience the web without ads today."
** toggle off state title: "Want Neeva to block ads, trackers and cookie popups on this page?"
* icons: [here're the latest graphics](https://www.figma.com/file/Qw2yhbLeFslwH4qYSu769i?node-id=0:1#268002131), lmk, if there're formats that I can export to.
* width for the toggle container and the cta button width: would it be possible to match the left and right ends? 


Thanks!!!

**Comment by chung-lai** [comment]
> Thanks @chung-lai !
> 
> Seems like the strings and the icons refer back to a few days older version. Here's the latest from Sep 13, i.e. latest & final:
> 
> * strings:
>   ** zero query subtext:  "Browse faster without spammy, screen-filling ads. Experience the web without ads today."
>   ** toggle off state title: "Want Neeva to block ads, trackers and cookie popups on this page?"
> * icon heights: when the states toggle on/off, the icon height (or top padding) seems to change. could you take a look?
> * icons: [here're the latest graphics](https://www.figma.com/file/Qw2yhbLeFslwH4qYSu769i?node-id=0:1#268002131), lmk, if there're formats that I can export to.
> * width for the toggle container and the cta button width: would it be possible to match the left and right ends?
> 
> Thanks!!!

Thanks @hwiatneeva!

1. will fix the strings
2. & 3. will fix the layout change and icon update in a follow up PR
3.  will try to match

File: Client/Frontend/Browser/CookieCutter/CookieCutterOnboardingView.swift [hunk 1]
```diff
@@ -33,11 +33,11 @@ struct CookieCutterOnboardingView: View {
                 ) {
                     VStack(alignment: .leading, spacing: 8) {
                         Text(
-                            "Your Cookie Cutter just declined a **cookie popup** and blocked \(trackingStatsViewModel.numTrackers) **trackers**"
+                            "Your Neeva Shield just declined a **cookie popup** and blocked \(trackingStatsViewModel.numTrackers) **trackers**"
                         )
 
                         Text(
-                            "Cookie Cutter by Neeva stops annoying cookie popups and blocks invasive trackers across the web."
+                            "Neeva Shield by Neeva stops annoying cookie popups and blocks invasive trackers across the web."
```
Comment (jedireza) [inline Client/Frontend/Browser/CookieCutter/CookieCutterOnboardingView.swift:40]:
```suggestion
                            "Neeva Shield stops annoying cookie popups and blocks invasive trackers across the web."
```

File: Client/Frontend/Settings/CookieCutter/CookieCutterSettings.swift [hunk 2]
```diff
@@ -17,10 +17,10 @@ struct CookieCutterSettings: View {
             Section(
                 footer:
                     Text(
-                        "When disabled, turns off Cookie Cutter on all sites. Changing this will reload your tabs."
+                        "When disabled, turns off Neeva Shield on all sites. Changing this will reload your tabs."
                     )
             ) {
-                Toggle("Cookie Cutter", isOn: $cookieCutterEnabled)
+                Toggle("Neeva Shield", isOn: $cookieCutterEnabled)
```
Comment (jedireza) [inline Client/Frontend/Settings/CookieCutter/CookieCutterSettings.swift:23]:
```suggestion
                Toggle("Neeva Shield", isOn: $neevaShieldEnabled)
```

Maybe change this variable name.

**Review by jedireza — APPROVED** [review]
Looking good. Left a couple code suggestions. Your call on those.

image

Curious if we should consider a spacer above the promo on the zero query screen for a more balanced feel.

**Comment by evancrow** [comment]
Couple of comments:

- It looks like there are two different designs for the shield menu; when is one or the other used? Is it based on if the user is signed in?
- "Neeva blocked cookie popup" sounds better as "Neeva blocked **a** cookie popup".

File: Client/Frontend/Browser/CookieCutter/CookieCutterOnboardingView.swift (hunk 1 above)
Comment (chung-lai) [reply Client/Frontend/Browser/CookieCutter/CookieCutterOnboardingView.swift:40]: make sense. cc @hwiatneeva

File: Client/Frontend/Settings/CookieCutter/CookieCutterSettings.swift (hunk 2 above)
Comment (chung-lai) [reply Client/Frontend/Settings/CookieCutter/CookieCutterSettings.swift:23]: > Couple of comments:
> 
> * It looks like there are two different designs for the shield menu; when is one or the other used? Is it based on if the user is signed in?
> * "Neeva blocked cookie popup" sounds better as "Neeva blocked **a** cookie popup".

Thanks for the feedback! What shield menu do you refer to?

`Neeva blocked a cookie popup` sounds good, will change!

**Pull request merged**